doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -237,39 +512,59 @@ Run WCS 1.0 tests
       * "0.1" for ``RESY``
 
    #. ``Options``:
-  
+
       * Ensure ``Verify that the server supports XML encoding`` is *checked*
       * Ensure ``Verify that the server supports range set axis`` is *checked*
 
    #. ``Schemas``:
 
-      * Ensure that ``original schemas`` is selected
+      * Ensure that ``The server implements the original schemas from the WCS 1.0.0 scpecification (OGC 03-065`` is selected
 
    #. Click ``OK``
 
-   .. image:: tewcs-1_0a.jpg
+   .. image:: ./image/tewcs-1_0a.png
 
-   .. image:: tewcs-1_0b.jpg
+   .. image:: ./image/tewcs-1_0b.png
 
-.. _commandline:
+   .. image:: ./image/tewcs-1_0c.png
 
-.. _teamengine:
-Teamengine Web Application
---------------------------
 
-The Teamengine web application is useful for analyzing results of a test run. To run the web application execute::
+Run WCS 1.1 tests
+-----------------
+
+#. Prepare the environment:
+
+  - Start GeoServer with the citewcs-1.1 data directory. Example:
+
+   .. code-block:: shell
+
+    cd <root of geoserver install>
+    export GEOSERVER_DATA_DIR=<root of geoserver sources>/data/citewcs-1.1
```

Comment:
Still points to the old, non existing directory.

Reply:
solved in https://github.com/randomorder/geoserver/pull/3/commits/91da4cbdc686ba5397014ab7887381ee240f8a1b